examples/add/package.json

```diff
@@ -1,9 +1,10 @@
 {
   "scripts": {
-    "build": "webpack",
-    "serve": "webpack-dev-server"
+    "build": "webpack -p",
+    "serve": "webpack-dev-server -p"
```

Comment:
I believe this is Webpack's default, we can omit it but I also like that's explicit.